app/src/main/java/com/github/onlynotesswent/ui/deck/DeckPlay.kt

```diff
@@ -310,39 +363,87 @@ private fun TestMode(
     isFinished: MutableState<Boolean>,
     answers: Map<String, MutableState<Int?>>,
 ) {
-  Column(
-      modifier = Modifier.testTag("TestModeColumn"),
-  ) {
+  Box(contentAlignment = Alignment.Center) {
     val pagerState = rememberPagerState { flashcardList.value.size }
-    HorizontalPager(pagerState, modifier = Modifier.testTag("Pager")) { pageIndex ->
-      Column(
-          modifier = Modifier.fillMaxWidth().padding(16.dp).testTag("flashcardColumn"),
-          horizontalAlignment = Alignment.CenterHorizontally,
-          verticalArrangement = Arrangement.spacedBy(16.dp)) {
-            val flashcardState = remember { derivedStateOf { flashcardList.value[pageIndex] } }
-            FlashcardPlayItem(
-                flashcardState,
-                fileViewModel,
-                onCorrect = { score.value += 1 },
-                choice = answers[flashcardList.value[pageIndex].id]!!)
+    val fling = PagerDefaults.flingBehavior(pagerState, snapPositionalThreshold = 0.3f)
+    HorizontalPager(
+        state = pagerState,
+        modifier =
+            Modifier.testTag("Pager")
+                .fillMaxWidth()
+                .heightIn(250.dp, 700.dp)
+                .align(Alignment.Center)
+                .padding(bottom = 80.dp),
+        flingBehavior = fling) { pageIndex ->
+          Column(
```

Comment:
This is very good and everything is properly shown on the screen, the little dots at the bottom are shown perfectly